docs/src/appendix/sncast/verify.md

```diff
@@ -6,7 +6,13 @@ Required.
 
 The address of the contract that is to be verified.
 
-## `--contract-name <CONTRACT_NAME>`
+
+## `--class-hash, -c <CONTRACT_ADDRESS>`
+Required.
+
+The address of the contract that is to be verified.
+
+## `--class-name <CONTRACT_NAME>`
```

Comment:
This is renamed as we feel its more accurate to call it class name rather than contract name. The class (or the code ) is what is being verified, and the contract is an instance of the class (contract). Thus why we decided to change the name to better reflect it.

Changing it to`<CLASS_NAME>` to make it clearer too.

Reply:
I wonder about this. I get your point about it verifying the class. However, we're using `--contract-name` in `sncast declare` even though it just declares the contract class, so this was named mostly for consistency.

Reply:
if that's the case then let's make it `--contract-name` for consistency. I think it all comes down to preferences, and we can wait and see if there's any feedback on this once its released.

Reply:
Sorry, just went through the specs and realized that changing this might not be a good idea since prior to this we had changed the specs and walnut had followed suit with the spec change where we use `--class-name` instead of `--contract-name`. Do we want to change this again? 

We can change it on our side and preserve the api payload schema, which should work. Wdyt @cptartur ?